wdl2cwl/tests/wdl_files/hisat2.wdl

```diff
@@ -25,7 +25,11 @@ task Hisat2 {
     input {
         File inputR1
         File? inputR2
+<<<<<<< HEAD
```

Comment:
You have some "merge conflict" markers left in the code 🙂 They should be removed